src/components/logo/AppLogoItem.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <polygon
+    class="app-logo-item"
+    :class="classes"
+    @click="$emit('click', $event)"
+    :points="points"
+    :fill="active ? color : 'rgba(255, 255, 255, 0.16)'"
+  >
+  </polygon>
+</template>
+
+<script lang="ts">
+import { defineComponent, computed } from '@vue/composition-api';
+export type Position = 'top' | 'right' | 'bottom' | 'left' | 'center';
```

Comment:
I think you don't need the export here

Reply:
Hmm... I think the problem is that I should be using the type from the consumer of this component.